sunpy/tests/figure_hashes_mpl_332_ft_261_astropy_42.json

```diff
@@ -1,6 +1,9 @@
 {
   "sunpy.map.tests.test_compositemap.test_plot_composite_map": "7d8ab4c0ffecd45de6e2829fa343bb289e107fd3ce4bcc8e7243673f7e472d99",
+  "sunpy.map.tests.test_compositemap.test_remove_composite_map": "67b17ac7c343a1d568d9dfa2bb12e0041e2fc53e8ca1c0e1c2c7693eda2bd0d8",
+  "sunpy.map.tests.test_compositemap.test_set_alpha_composite_map": "67b17ac7c343a1d568d9dfa2bb12e0041e2fc53e8ca1c0e1c2c7693eda2bd0d8",
```

Comment:
Should these have the same hash?

Reply:
These figure tests don't actually plot anything, so the generated figure in each case is simply a blank figure, hence why the hashes are the same.

Reply:
Ah, I was wondering why the hash looked so familiar to me.